### PR TITLE
Support learning rate scheduler

### PR DESCRIPTION
**Pull request by skydoorkai**
The user can optionally provide a learning rate scheduler method, which takes a model version as its input,  and returns a corresponding learning rate value.

### CODE REVIEW

File: elasticdl/python/common/lr_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import threading
+
+
+class LearningRateScheduler:
+    def __init__(self, learning_rate=0.001, model_version=0):
```
Comment (workingloong) [inline elasticdl/python/common/lr_scheduler.py:5]: Do we need to support scheduling the learning rate by epoch?

**Review by QiJune — APPROVED** [review]
Excellent!

File: elasticdl/python/common/lr_scheduler.py (hunk 1 above)
Comment (skydoorkai) [reply elasticdl/python/common/lr_scheduler.py:5]: PS does not know how many batches in an epoch. The user knows it and can set the scheduler accordingly.
